auth/store.go

```diff
@@ -807,20 +879,53 @@ func (as *authStore) isAuthEnabled() bool {
 	return as.enabled
 }
 
-func NewAuthStore(be backend.Backend) *authStore {
+func NewAuthStore(be backend.Backend, tokenType, jwtSignMethod, jwtPubKeyPath, jwtPrivKeyPath string) *authStore {
```

Comment:
Because of the same reason to this comment (https://github.com/coreos/etcd/pull/6082#discussion_r76182729), I think separating the init functions wouldn't be good for now.